circle.yml

```diff
@@ -1,5 +1,8 @@
 version: 2.1
 
+orbs:
+  shellcheck: circleci/shellcheck@1.3.16
```

Comment:
This is actually just a pip package, we should include it in `cpp-build-env` and make it part of the lint step.

Reply:
Not a pip package, but added to cpp-build-env:15-lint and cpp-build-env:15-clang-10.